src/Constraint/Factory/FactoryTrait.php

```diff
@@ -4,15 +4,14 @@
 
 namespace JBen87\ParsleyBundle\Constraint\Factory;
 
+use Symfony\Contracts\Service\Attribute\Required;
 use Symfony\Contracts\Translation\TranslatorInterface;
 
 trait FactoryTrait
 {
     private TranslatorInterface $translator;
 
-    /**
-     * @required
-     */
+    #[Required]
```

Comment:
Something like that:
```suggestion
    /**
     * @required
     */
    #[Required]
```